index.ts

```diff
@@ -82,13 +82,77 @@ function makeHash(viewFunction: string): string {
     return createHash("MD5").update(viewFunction).digest("hex");
 }
 
+export interface ViewFunctionConfig {
+    viewFunction: string;
+    model?: any;
+    decoder?: string;
+    indent?: number;
+    newLines?: boolean;
+}
+
+// compiles multiple view functions into one elm file
+// which is much faster if you're likely to need all of them
+export function grouped(
+    rootDir: string, moduleName: string, configs: ViewFunctionConfig[],
+    alreadyRun?: boolean, elmMakePath?: string, installMethod?: string): Promise<string[]> {

```

Comment:
💭 I wonder if returning `Promise<string, string>[]` would be better: that way it wouldn't block on all of them finishing

Reply:
Does this have any benefit when the elm app isn't truly parallel? 

You could send in the input via ports, which in theory could give parallelism, but elm doesn't currently support that.  Instead you could spawn the elm app multiple times, but that probably comes with considerable overhead.

But maybe when you have enough models with a small number of view functions, that's worth it?